test/unit/store/item-store.js

```diff
@@ -82,10 +82,25 @@ describe('ItemStore', () => {
         });
     });
 
+    it('should check out a single item', () => {
+        return addAction('NEW_CHECKOUT', {
+            studentId: student.id,
+            itemAddresses: [items[0].address]
+        }).then(() => {
+            assert.strictEqual(items[0].status, 'CHECKED_OUT');
+            assert.strictEqual(items[1].status, 'AVAILABLE');
+        });
+    });
+
     it('should check an item in', () => {
-        return addAction('CHECKIN', {
+        return addAction('NEW_CHECKOUT', {
```

Comment:
I added this because this test failed when I added [this test](https://github.com/TheFourFifths/consus/pull/52/commits/52b3c6713943ae6dfe6ceff0b8140269676e6183#diff-3dd376d611ae34cb10e2c491336e5f26R193).
I also added the above test to clarify the cause of this test failing iff `addAction('NEW_CHECKOUT', ...)` fails in _this_ test.